thoth_user_api/swagger.yaml

```diff
@@ -553,6 +553,38 @@ definitions:
           Collecting tensorflow
           Downloading tensorflow-1.4.1-cp36-cp36m-manylinux1_x86_64.whl (41.2MB)
           Successfully installed tensorflow-1.4.1
+      apiVersion:
+        type: string
+        description: >
+          BuildLog api version.
+        example: >
+          "apis/build.openshift.io/v1/namespaces/$NAMESPACE/builds HTTP/1.1"
+      kind:
+        type: string
+        description: >
+          Type of log.
+        example: >
+          "BuildLog"
+      metadata:
+        type: string
+        description: >
+          meta data artifacts of the build config.
+        example:
+          "annotations"
+        properties:
+          annotations:
+            type: string
+            description: >
+             All the BuildConfiguration of the build.
+            example: >
+              "buildlog.thoth-core/build-config"
+            properties:
+              buildlog.thoth-station.ninja/corresponding-build-configuration:
+                type: string
+                description: >
+                 Contains all particulars about the build log.
+                example: >
+                "apiVersion": "build.openshift.io/v1"
```

Comment:
This example for type string looks invalid.

BTW you can run API server with the following command to ensure the Swagger specification is valid and connexion can parse your configuration (the calls will not result to calls of actual endpoint handlers):

```console
pipenv run connexion run thoth_user_api/swagger.yaml --stub --debug
```